{{cookiecutter.project_slug}}/src/features/{% if cookiecutter.has_camera_blueprint == "y" %}UserCamera{% endif %}/screens/index.js

```diff
@@ -0,0 +1,44 @@
+import React, {PureComponent} from 'react';
+import {View, Platform, StyleSheet} from 'react-native';
+import {Text, Button} from 'react-native-ui-kitten';
+
+export default class Root extends PureComponent {
+  static navigationOptions = {
+    title: 'Camera'.toUpperCase(),
+    headerBackTitle: null,
+  };
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Button
+          onPress={() => this.props.navigation.navigate('Camera')}
+          style={styles.item}>
+          Camera
+        </Button>
+
+        <Button
+          onPress={() => this.props.navigation.navigate('SplashScreen')}
+          style={styles.item}>
+          Main Menu
+        </Button>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Since I see that we are repeating this in all the HomeScreens, why don't we move it the general style file located in `src/config/crowdboticsTheme` ?